duckduckgo/duckduckgo-autofill
# Set tooltip initial offscreen position

**Reviewer:** @GioSensation 
**Asana:** https://app.asana.com/0/1200930669568058/1204259127711851/f

## Description
Sets the initial position of the tooltip to be far offscreen before the `hidden` attribute is removed.

## Steps to test
1. Go to https://69bboutique.com/
2. Click on Dax icon to open tooltip -- watch the top left corner to see if there's a flash of tooltip
3. Repeat as needed
